File: 674 - Longest Continuous Increasing Subsequence.java
class Solution {
    public int findLengthOfLCIS(int[] nums) {
        int maxsk = 1, cursk = 1;
        if (nums.length == 0) {
            return 0;
        }
        for (int i = 0; i < nums.length - 1; i++) {
            if (nums[i] < nums[i + 1]) {
                cursk++;
            } else {
                maxsk = Math.max(cursk, maxsk);
                cursk = 1;
            }

        }
        return Math.max(cursk, maxsk);
    }
}

File: 674 - Longest Continuous Increasing Subsequence.py
class Solution:
    def findLengthOfLCIS(self, nums: list[int]) -> int:
        if len(nums) == 0:
            return 0
        consk = 1
        maxsk = 1
        for i in range(len(nums) - 1):
            if nums[i] < nums[i + 1]:
                consk += 1
            else:
                maxsk = max(maxsk, consk)
                consk = 1
        return max(maxsk, consk)
